ssas/groups.go

```diff
@@ -66,6 +68,58 @@ func CreateGroup(gd GroupData) (Group, error) {
 	return g, nil
 }
 
+func UpdateGroup(id string, gd GroupData) (Group, error) {
```

Comment:
If we're using the database ID as the group identifier for API calls (which is ok for uniqueness), we'll need to make sure we're returning it in `CreateGroup()`.  Right now that returns a 0 for the Group.ID.

Reply:
createGroup now returns the the created group so that the id can be used